client/src/client.rs

```diff
@@ -424,16 +432,18 @@ impl Client {
         let response = req
             .build()?
             .send()
-            .wrap_err_with(|| format!("Failed to send transaction with hash {:?}", hash))?;
+            .wrap_err_with(|| format!("Failed to send transaction with hash {hash}"))?;
```

Comment:
```suggestion
            .wrap_err_with(|| format!("Failed to send transaction with hash {hash:?}"))?;
```

I guess `Debug` output is more suitable in an error branch.